src/pages/Flow/Run.vue

```diff
@@ -444,6 +465,53 @@ export default {
           </v-col>
         </v-row>
 
+        <v-row v-if="showAdvanced" class="my-2 py-8 row-divider" no-gutters>
+          <v-row class="my-2 py-8" no-gutters>
+            <v-col cols="12" md="3">
+              <div
+                class="py-0"
+                :class="{ 'pr-24': $vuetify.breakpoint.mdAndUp }"
+              >
+                <div class="text-h5">
+                  Logging Level
+                  <MenuTooltip>
+                    <p>
+                      Logging level for the flow run. If none is specified,
+                      default agent logging level will be used.
+                    </p>
+                    <p>
+                      This can also be controlled by providing an environment
+                      variable, "PREFECT__LOGGING__LEVEL".
+                    </p>
+                    <p>
+                      Any dropdown selection besides "Default" will override the
+                      environment variable.
+                    </p>
+                    <p>
+                      Please note this is only guaranteed to work for Agents
+                      running Prefect Core TODO or later.
+                    </p>
+                  </MenuTooltip>
+                </div>
+              </div>
+            </v-col>
+
+            <v-col cols="12" md="9" class="mt-n4 mt-md-0">
+              <v-select v-model="loggingLevel" outlined :items="loggingLevels">
+                <template #item="{ item }">
+                  <div>
+                    <v-chip
+                      :class="['ma-2', 'debuglevel', item.text.toLowerCase()]"
```

Comment:
pretty sure this would be more legible as 
```
class="ma-2 debuglevel"
:class="item.text.toLowerCase"
```

(I'm not 100% on my syntax there, might still need the parens on toLowerCase, so I'd test before taking this suggestion wholesale)

Reply:
That is very cool!

I'm having some trouble getting it to work though, is it worth setting aside some time to pair?

Reply:
oh! I misunderstood what you were doing here. try `class="ma-2 debuglevel" :class="item.text.toLowerCase()"` (I read it initially as you were trying to make the words lowercase, not that you were adding a classname and making sure _that_ was cased properly. sorry!)

Reply:
Ahhh that makes sense now. Still cool!

Updated the syntax in my latest commit